src/sidebar/sidebar.js

```diff
@@ -0,0 +1,60 @@
+import { CustomDataProvider } from '@dhis2/app-runtime'
+import PropTypes from 'prop-types'
+import React from 'react'
+import AuditLog from './audit-log.js'
+import BasicInformation from './basic-information.js'
+import Comment from './comment.js'
+import ItemPropType from './item-prop-type.js'
+import Limits from './limits.js'
+import styles from './sidebar.module.css'
+import Title from './title.js'
+
+const mockData = {
+    comment: 'Some comment',
+    limits: {
+        avg: 238,
+        min: 50,
+        max: 250,
+    },
+    auditLog: [
+        {
+            displayName: 'Firstname Lastname',
+            changeType: 'UPDATE',
+            newValue: '19',
+            oldValue: '13',
+            at: new Date('2022-01-01'),
+        },
+        {
+            displayName: 'Firstname Lastname',
+            changeType: 'DELETE',
+            oldValue: '15',
+            at: new Date('2021-10-01'),
+        },
+    ],
+}
```

Comment:
Like the data item schema, this is how I imagined the schemas for the various data item related resources based on the requirements of the design spec.